.taskman_functions

```diff
@@ -0,0 +1,100 @@
+#this is a dot file which will contain the taskman functions 
+
+#Usage: add_task "Task description"
+add_task() {
+ 
+    #I used local here to make it only function specific scope
+    local description=$1
+
+    if [[ -z "$description" ]]; then
+        echo "Kindly add the task description"
+        return 1  
+    fi
+
+    #we then generated a new task id by substituting our 2 operations and appending 1
+    local id=$(($(tail -n 1 "$TASKMAN_DIR/tasks.txt" | cut -d'|' -f1) + 1))
+    id=${id:-1}
+    echo "$id|Pending|$description" >> "$TASKMAN_DIR/tasks.txt"
+    echo "Task is successfully added: $description"
+}
+
+#Displays all tasks in the following format:
+#ID | Status | Task
+#+++++++++++++++++++++++++++++++++++++++
+#1 | Pending | Finish the report
+#2 | Done | Review the code
+list_tasks() {
+
+    if [ ! -s "$TASKMAN_DIR/tasks.txt" ]; then
+        echo "No tasks are added."
+        return
+    fi
+
+    echo "ID | Status  | Task" #first of all we print the header as in the question
+    echo "+++++++++++++++++++++++++++++++++++++++" 
+    #Then we open the file and format it 
```

Comment:
Commenting in this tone is not okay. "Then" tells me that there is a continuation of comments. However I believe a comment should just be about a following statement. 

Also, if the line explains it well enough, there is no need for a comment. 

Another point is, please put a space between # and the actual comment line. No need to always put first letter as a capital for 1 line comments.

Reply:
I will look into it, rectify it and remember this for future PRs.